tests/data/leaf_axis.drf

```diff
@@ -1,9 +1,9 @@
-SMBPath = D:/Documents/pradal/devlp/vplants/aml/databases/SMBFiles
-#SMBPath = ../../../databases/SMBFiles
-
-SMBModel feuille = feui136 #feui113
-
-# Geometry = ../../../databases/GEOMFiles/leaf.geom
-# contains symbol a_leaf
-
-Class F = feuille
+SMBPath = D:/Documents/pradal/devlp/vplants/aml/databases/SMBFiles
```

Comment:
PATh are only valid on windows on a specific computer. Make this system independent

Reply:
Where should be located this folder (SMBFiles) ? It does not appear to be in the git repo